packages/fxa-content-server/server/lib/routes/validate-email-domain.js

```diff
@@ -68,14 +67,18 @@ module.exports = function () {
     },
     process: async function (req, res, next) {
       const { domain } = req.query;
-
+      const value = config.get('mxRecordValidation');
+      console.log(value);
```

Comment:
Please remove the debug logging.  Also, let's name the variable with something less generic.  Maybe `shouldValidate` or something.